plugin-server/src/utils/db/db.ts

```diff
@@ -149,19 +149,29 @@ export class DB {
     /** Whether to write to clickhouse_person_unique_id topic */
     writeToPersonUniqueId?: boolean
 
+    /** How many seconds to keep person info in Redis cache */
+    PERSON_INFO_CACHE_TTL: number
+
+    /** Which teams is person info caching enabled on */
+    personInfoCachingEnabledTeams: Set<number>
+
     constructor(
         postgres: Pool,
         redisPool: GenericPool<Redis.Redis>,
         kafkaProducer: KafkaProducerWrapper | undefined,
         clickhouse: ClickHouse | undefined,
-        statsd: StatsD | undefined
+        statsd: StatsD | undefined,
+        personInfoCacheTtl = 1,
```

Comment:
would be best to have this to the same default as the config but sure

Reply:
set it like that on purpose to expire fast if not set.